config/nrfconnect/chip-module/zephyr/module.yml

```diff
@@ -14,6 +14,7 @@
 #   limitations under the License.
 #
 
+name: connectedhomeip
```

Comment:
Could you also add "depends: -openthread" part from the sdk-connectedhomeip manifest? I noticed that it's necessary to make sure that NCS-supplied CHIP is replaced with the current CHIP.

Reply:
Ok, I had to restore `zephyr_include_directories(${ZEPHYR_BASE}/../nrfxlib/crypto/nrf_cc310_platform/include)` because after that crypto headers were missing, so for now it has to stay.